tests/common/plugins/custom_markers/__init__.py

```diff
@@ -51,7 +52,9 @@ def check_topology(item):
         if all(topo not in toponames.args for topo in cfg_topos):
             pytest.skip("test requires topology in {!r}".format(toponames))
     else:
-        pytest.skip("testcase {} is skipped when no topology marker is given".format(item.name))
+        warn_msg = "testcase {} is skipped when no topology marker is given".format(item.name)
```

Comment:
Suggest to use item.nodeid here. Comparing with the `name` property, the `nodeid` property contains full information of the test case.

Reply:
Thanks! I've made a update.